tasks/main.yml

```diff
@@ -15,32 +15,61 @@
     repo: '{{ docker__upstream_repository }}'
     state: 'present'
     update_cache: True
+  register: docker__register_apt_repo
```

Comment:
yes, you are right. I missed this one. I think it is a left over from when I wanted to make the role capable of switching between stable and edge. I'll remove it.